### PR TITLE
feat: Add collapse option to the read activity (regular or kudos activity) - MEED-4574 - Meeds-io/MIPs#124

### PR DESCRIPTION
**Pull request by SaraBoutej**
Prior to this PR, the read activity content was fully displayed. 
This change allows for the addition of a collapsed option to the activity content extension, displaying only the first 4 lines of the content with a 'read more' option to view the full content.

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue [hunk 1]
```diff
@@ -252,6 +273,12 @@ export default {
   },
   created() {
     this.retrieveActivityProperties();
+    window.addEventListener('resize', () => {
+      this.displayReadMore();
+    });
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue]:
```suggestion
    window.addEventListener('resize', this.displayReadMore);
  },
  beforeDestroy() {
    window.removeEventListener('resize', this.displayReadMore);
```

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityBody.vue [hunk 2]
```diff
@@ -68,14 +88,25 @@ export default {
   },
   created() {
     this.retrieveActivityProperties();
+    window.addEventListener('resize', () => {
+      this.displayReadMore();
+    });
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityBody.vue]:
```suggestion
    window.addEventListener('resize', this.displayReadMore);
  },
  beforeDestroy() {
    window.removeEventListener('resize', this.displayReadMore);
```

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue (hunk 1 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue]: fixed

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityBody.vue (hunk 2 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityBody.vue]: fixed

**Review by boubaker — APPROVED** [review]

**Pull request merged**